cocos/2d/framework/ui-renderer.ts

```diff
@@ -231,8 +234,11 @@ export class UIRenderer extends Renderer {
     @type(Material)
     protected _customMaterial: Material | null = null;
 
+    /**
+     * @engineInternal
+     */
     @serializable
-    protected _srcBlendFactor = BlendFactor.SRC_ALPHA;
+    public _srcBlendFactor = BlendFactor.SRC_ALPHA;
```

Comment:
Why change to public?

Reply:
I think it is better:
- make it private
- add getter/setter as needed, and make them as engine internal

Reply:
updated https://github.com/cocos/cocos-engine/pull/14572/commits/018e80e6f0d06893f403aeadbdf60703e065a01c